eslint-config/package.json

```diff
@@ -6,11 +6,12 @@
   "main": "index.js",
   "scripts": {},
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
-    "babel-eslint": "^10.1.0",
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb": "^18.2.1",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/preset-react": "^7.16.7",
```

Comment:
Do we really need `@babel/preset-react` in the `eslint-config`? I'm not sure but I think this is related to compilation rather than linting and maybe we need only the parser here.

(We could also try if the default eslint's parser works and if it's faster or slower but yeah, not needed now.)

Reply:
Addressed in 2f65af2473aa58b19175f37b6d5a138fce748011